src/modules/storage/LedgerStorage.ts

```diff
@@ -1318,4 +1318,42 @@ export class LedgerStorage extends Storages
 
         return this.query(sql, [address]);
     }
+
+    /**
+     * Gets a transaction data
+     * @param tx_hash The hash of the transaction to get
+     * @returns Returns the Promise. If it is finished successfully the `.then`
+     * of the returned Promise is called with the records
+     * and if an error occurs the `.catch` is called with an error.
+     */
+    public getTransactionPending (tx_hash: Hash): Promise<Transaction | null>
+    {
+        return new Promise<Transaction | null>(async (resolve, reject) =>
+        {
+            try {
+                let hash = tx_hash.toBinary(Endian.Little);
+                let rows = await this.query("SELECT * FROM transaction_pool WHERE tx_hash = ?;", [hash]);
```

Comment:
You must define the exact query column.
This may include data that is unnecessary or should not be provided.